ERC821.md

```diff
@@ -75,7 +75,7 @@ Return the total amount of assets under this DAR. This method SHALL NOT throw.
 
 #### `supportsInterface(bytes8 interfaceId):bool`
 
-This method returns `true` if the `interfaceId` is a supported interface (165, corresponding to 0x01ffc9a7, or 821, corresponding to 0x959d7abb). This method SHALL NOT throw.
+This method returns `true` if the `interfaceId` is a supported interface (165, corresponding to 0x01ffc9a7, or 821, corresponding to 0x7c0633c6). This method SHALL NOT throw.
```

Comment:
`0x80ac58cd`

Reply:
I think just changing the docs here might do the trick?

```
This method returns `true` if the `interfaceId` is a supported interface (165, corresponding to 0x01ffc9a7, 821, corresponding to 0x7c0633c6, or 721 corresponding to 0x80ac58cd). This method SHALL NOT throw.
```

Reply:
agree